dev/CommonStyles/TextBox_themeresources.xaml

```diff
@@ -73,11 +75,12 @@
             <SolidColorBrush x:Key="TextBoxDisabledBorderThemeBrush" Color="#26000000" />
             <SolidColorBrush x:Key="TextBoxDisabledForegroundThemeBrush" Color="#FF666666" />
             <SolidColorBrush x:Key="TextBoxForegroundThemeBrush" Color="#FF000000" />
-            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemAltHighColor" />
+            <StaticResource x:Key="TextControlBackgroundFocused" ResourceKey="SystemControlBackgroundAltHighBrush" />
             <StaticResource x:Key="TextControlBorderBrush" ResourceKey="SystemControlForegroundBaseMediumBrush" />
             <StaticResource x:Key="TextControlBorderBrushPointerOver" ResourceKey="SystemControlHighlightBaseMediumHighBrush" />
-            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemBaseHighColor" />
-            <StaticResource x:Key="TextControlForegroundFocused" ResourceKey="SystemBaseHighColor" />
+            <StaticResource x:Key="TextControlButtonForeground" ResourceKey="SystemControlPageTextBaseHighBrush" />
```

Comment:
Better to use `SystemControlForegroundBaseMediumHighBrush` to match the original.